src/protagonist/Test.Helpers/Integration/DlcsDatabaseFixture.cs

```diff
@@ -64,7 +64,7 @@ public void CleanUp()
         DbContext.Database.ExecuteSqlRaw("DELETE FROM \"EntityCounters\" WHERE \"Type\" = 'space' AND \"Customer\" != 99");
         DbContext.Database.ExecuteSqlRaw("DELETE FROM \"EntityCounters\" WHERE \"Type\" = 'space-images' AND \"Customer\" != 99");
         DbContext.Database.ExecuteSqlRaw("DELETE FROM \"EntityCounters\" WHERE \"Type\" = 'customer-images' AND \"Scope\" != '99'");
-        DbContext.Database.ExecuteSqlRaw("DELETE FROM \"DeliveryChannelPolicies\" WHERE \"Customer\" <> 1");
+        DbContext.Database.ExecuteSqlRaw("DELETE FROM \"DeliveryChannelPolicies\" WHERE \"Customer\" not in ('1','99')");

```

Comment:
`Customer` is an `int` so using `not in ('1', '99');` would mean postgres does an implicit conversion, which should be fine but generally better to use type if you know it (so `not in (1, 99);`)